src/components/Input.js

```diff
@@ -0,0 +1,13 @@
+import PropTypes from 'prop-types';
+
+function Input({ type, style, placeholder }) {
+  return (
+    <input style={style} type={type} placeholder={placeholder} />
+  );
+}
+Input.propTypes = {
+  type: PropTypes.string.isRequired,
+  placeholder: PropTypes.string.isRequired,
+  style: PropTypes.string.isRequired,
+};
```

Comment:
- [ ] Instead of creating an input component for individual inputs please create a `<NewBookForm>` component which
- Renders a form with inputs for the title and the author of the book
- Has a button to submit
- the button wouldn't add the book *yet*, this is just a placeholder for now